### ISSUE TITLE
Code checkin to Layer5

### ISSUE THREAD

**Issue by kumarabd** [Aisuko/helm-charts-action#2]
Hey @Aisuko , Can this be moved to - https://github.com/layer5io/helm-chart-releaser, so that it is appropriate to be used for Layer5 projects.

**Comment by Aisuko** [comment]
Of course, there is no problem, the repo already transferred to the community https://github.com/layer5io/chart-releaser-action-old, and I will add the content to the new repo which I saw you created after fixed the issue.
